test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -14,6 +11,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
+import seedu.addressbook.util.TestUtil;
 
```

Comment:
Is the newline here intentional?

Reply:
Yes, it's to separate the import statements into different blocks for readability